simulator-gradle-plugin/src/main/kotlin/org/robolectric/simulator/gradle/SimulatorPlugin.kt

```diff
@@ -87,7 +86,7 @@ class SimulatorPlugin : Plugin<Project> {
       jvmArgs = testTask.jvmArgs + robolectricJvmArgs
       mainClass.set(MAIN_CLASS)
       args = listOf(resourceApkFile.absolutePath)
-      dependsOn(testTaskName, "assembleDebug")
+      dependsOn("assembleDebug")
```

Comment:
IIUC, this simulate task requires `debutTest` task finished?

Reply:
What I know is that "apk" in test environment is liek "xxx._ap", and it is generated by test task. If you we remove this task dependency, how can we ensure the "apk" file is generated ?

Reply:
Ah, I thought it was looking for the app APK, not the test APK.

So maybe, `dependsOn` should include the following?
https://github.com/robolectric/robolectric/blob/7a3b64d574dcd92bddd29e031296b8ca8e86be07/simulator-gradle-plugin/src/main/kotlin/org/robolectric/simulator/gradle/SimulatorPlugin.kt#L45-L46

`dependsOn(targetTask, "assembleDebug"`

Reply:
The task exists, but it doesn't seem to generate the necessary file. I'm looking to see if I can find an other one